frontend/lib/norent/homepage.tsx

```diff
@@ -79,19 +81,21 @@ export const LandingPageChecklist = (props: {
     <div className="hero-body">
       <div className="container jf-has-text-centered-tablet">
         <h3 className="is-size-5 is-spaced has-text-weight-normal">
-          Here’s what you can do with <NorentLogo size="is-128x128" />
+          <Trans>
+            Here’s what you can do with <NorentLogo size="is-128x128" />
+          </Trans>
```

Comment:
Again, I wonder if a `comment` attribute would help the translator make better sense of this?  Or on the other hand, Crowdin does make the file and line number visible to the translator, but it doesn't seem to have any way of providing a hyperlink directly to the location on GitHub... it would be cool if it did, at least then the translator could click on it to see the `<NorentLogo>`.

Reply:
OH WAIT!  You know what might be awesome?!! If this JSX actually read like this:

```jsx
<Trans>
  Here's what you can do with <NorentLogo size="is-128x128">NoRent</NorentLogo>
</Trans>
```

The `<NorentLogo>` could simply not render its children, but instead use the children as its `alt` text.  And this would show up to translators as something like:

```
Here's what you can do with <0>NoRent</0>
```

That would be a lot easier for them to understand than the current code, which I think would read to translators as:

```
Here's what you can do with <0>
```

it would also allow them to easily change the alt text I guess, although I'm not sure if that's desirable or not.

Anyways just an idea.

Reply:
Err if you actually think this idea is good, I would hold on actually implementing it for another PR though...

Reply:
That's a really good idea!! Ok ill save this for another PR to work on next week.